Loaded module: cuda/11.8
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
Missing logger folder: tensorboard_logs/mse_with_print
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name          | Type                    | Params
----------------------------------------------------------
0 | loss_function | MSELoss                 | 0     
1 | img_encoder   | ResNet50                | 24.0 M
2 | R_encoder     | TransformerTextEmbedder | 16.0 M
3 | W_R           | Linear                  | 65.8 K
4 | W_img         | Linear                  | 65.8 K
5 | accuracy      | MulticlassAccuracy      | 0     
----------------------------------------------------------
16.7 M    Trainable params
23.5 M    Non-trainable params
40.2 M    Total params
160.756   Total estimated model params size (MB)
/work3/s184984/repos/ADLCV-recipe-retrieval/recipe/lib/python3.11/site-packages/lightning/pytorch/trainer/connectors/data_connector.py:441: The 'val_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=5` in the `DataLoader` to improve performance.
/work3/s184984/repos/ADLCV-recipe-retrieval/recipe/lib/python3.11/site-packages/lightning/pytorch/trainer/connectors/data_connector.py:441: The 'train_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=5` in the `DataLoader` to improve performance.
/work3/s184984/repos/ADLCV-recipe-retrieval/recipe/lib/python3.11/site-packages/lightning/pytorch/trainer/call.py:54: Detected KeyboardInterrupt, attempting graceful shutdown...
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
/work3/s184984/repos/ADLCV-recipe-retrieval/recipe/lib/python3.11/site-packages/lightning/pytorch/trainer/connectors/data_connector.py:441: The 'test_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=5` in the `DataLoader` to improve performance.
[rank: 0] Received SIGTERM: 15
[rank: 0] Received SIGTERM: 15
Terminated
Loaded module: cuda/11.8
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name          | Type                    | Params
----------------------------------------------------------
0 | loss_function | MSELoss                 | 0     
1 | img_encoder   | ResNet50                | 24.0 M
2 | R_encoder     | TransformerTextEmbedder | 16.0 M
3 | W_R           | Linear                  | 65.8 K
4 | W_img         | Linear                  | 65.8 K
5 | accuracy      | MulticlassAccuracy      | 0     
----------------------------------------------------------
16.7 M    Trainable params
23.5 M    Non-trainable params
40.2 M    Total params
160.756   Total estimated model params size (MB)
/work3/s184984/repos/ADLCV-recipe-retrieval/recipe/lib/python3.11/site-packages/lightning/pytorch/trainer/connectors/data_connector.py:441: The 'val_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=5` in the `DataLoader` to improve performance.
/work3/s184984/repos/ADLCV-recipe-retrieval/recipe/lib/python3.11/site-packages/lightning/pytorch/trainer/connectors/data_connector.py:441: The 'train_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=5` in the `DataLoader` to improve performance.
